curvefs/src/client/common/config.cpp

```diff
@@ -308,6 +308,35 @@ void InitFuseClientOption(Configuration *conf, FuseClientOption *clientOption) {
         clientOption->entryTimeOut = 0;
     }
 
+
+    conf->GetValueFatalIfFail("fuseClient.throttle.avgWriteBytes",
+                              &clientOption->avgWriteBytes);
```

Comment:
Here you can consider initializing FLAGS_XX directly, without going through the option to switch to one layer.

Reply:
I referred to the qos usage of S3 disk cache。

Reply:
There is a situation where users only need to initialize from the configuration file and do not need to dynamically modify the qos

Reply:
If so, the following regular InitQosParam will be problematic.

Even if the user does not set the value of FLAG_XX, the value read from the configuration file will be overwritten.

Reply:
> There is a situation where users only need to initialize from the configuration file and do not need to dynamically modify the qos

It can be considered that FLAG_XX is directly read in the code, and there are two sources of its assignment, one is the configuration file, and the other is the setting of the http interface.

Reply:
The fuse client initialization will read from the configuration file and assign values to FLAG_XX.
![1684325977(1)](https://github.com/opencurve/curve/assets/19617847/4f87f418-6247-4691-88ec-bab0a4406103)

Reply:
There are two ways to initializing values . Is this the standard writing method in the future? （initializing FLAGS_XX directly)